Cargo.toml

```diff
@@ -6,21 +6,31 @@ edition = "2021"
 license = "MIT OR Apache-2.0"
 
 [dependencies]
-hal = { package = "{{ mcu }}-hal", version = "{{ hal_version }}" }
+{{ mcu }}-hal = "{{ hal_version }}"
```

Comment:
What's the reason we don't want to alias the HAL dependency anymore?

Reply:
What was the reason we did it in the first place? I always found it weird and needless.

Reply:
Oh ... interesting. I found it to be easier to copy code around this way ... however, I don't have a hard preference. Was just curious - and also once we did the split, we will just have `esp_hal` or something

Reply:
I don't feel too strongly about this, so feel free to revert if anybody else does.

Reply:
I dont have any hard preference, tbh

Reply:
The main idea behind alias is to increase portability of the code. Using alias makes it easier to have common app code for multiple targets.

Reply:
The next release will unify the hal, meaning all hals will be exposed under `esp_hal`, just like `esp_idf_hal` supports all target in one.